website/content/en/docs/concepts/nodepools.md

```diff
@@ -233,6 +233,10 @@ Karpenter prioritizes Spot offerings if the NodePool allows Spot and on-demand i
 
 Karpenter also allows `karpenter.sh/capacity-type` to be used as a topology key for enforcing topology-spread.
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
This shouldn't be in the `docs` directory since it's not up-to-date with the latest release. If we want to put this in the docs for the current version, then it should be `30` since that's what the current restriction is

Reply:
Basically, any new updates that are going to come out at the next version should be in the `preview` directory so that we properly stage them out

Reply:
Updated